tutor/src/screens/assignment-edit/ux.js

```diff
@@ -422,4 +432,12 @@ export default class AssignmentUX {
   @action.bound onHideAddTemplate() {
     this.isShowingAddTemplate = false;
   }
+
+  @action setDidUserChangeDatesManually(didChange) {
+    this.didUserChangeDatesManually = didChange;
+  }
+
+  @action setDueAt(dueAt) {
+    this.dueAt = dueAt;
+  }
 }
```

Comment:
do we need the setter methods? Seems like they're not doing anything that couldn't be achieved by setting the values directly.